### ISSUE TITLE
failure following the Raspberry Pi OS armhf installation instructions

### ISSUE THREAD

**Issue by bfallik** [containers/podman.io_old#336]
Hi,

I've been trying to install Podman 2.2 according to the [installation instructions](https://podman.io/getting-started/installation) but so far I've been unsuccessful. Here are the current instructions:

```
# Raspbian 10
# Use buster-backports on Rasbian 10 for a newer libseccomp2
echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
echo 'deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Raspbian_10/ /' | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Raspbian_10/Release.key | sudo apt-key add -
sudo apt-get update -qq
sudo apt-get -qq -y install podman
```

I've hit 2 different issues.

Neither of my rpis had the signing key for buster-backports available. `apt` complained when I tried to update the cache:

```
Err:1 http://deb.debian.org/debian buster-backports InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 04EE7237B7D453EC NO_PUBKEY 648ACFD622F3D138
```

I was able to workaround this problem by first manually installing [debian-archive-keyring_2019.1_all.deb](http://deb.debian.org/debian/pool/main/d/debian-archive-keyring_2019.1_all.deb). It might be useful to include that step in the instructions.

Once the backports are set up I still can't install podman 2.2:

```
# apt-get install podman
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 podman : Depends: libseccomp2 (>= 2.4.3-1) but 2.3.3-4 is to be installed
E: Unable to correct problems, you have held broken packages.
```

I'm not sure where to look for the proper version of libseccomp2.

**Comment by rhatdan** [comment]
@lsm5 @Luap99 PTAL

**Comment by bfallik** [comment]
Hi, just wanted to check back in on this issue. I retried to install podman 2.2. and encountered the same issue with libseccomp2. Anyone have updated instructions for Raspbian?

Thanks,
brian

**Comment by justinschuldt** [comment]
Hey @bfallik, I also encountered that `public key is not available` error. I installed a package to fix that, but I'm not sure it is the best solution: `sudo apt-get install debian-archive-keyring`

Regarding the unmet libseccomp2 dependency, check out this PR https://github.com/containers/podman.io/pull/343. You just have have to run `sudo apt-get -y -t buster-backports install libseccomp2` to manually install the backport version of the package before installing podman.

**Comment by bfallik** [comment]
thanks @justinschuldt , that additional step worked to install libseccomp2 and unblock my podman upgrade:

```
# podman version
Version:      3.0.0-rc1
API Version:  3.0.0
Go Version:   go1.15.2
Built:        Wed Dec 31 19:00:00 1969
OS/Arch:      linux/arm
```

Looks like I was unexpectedly upgraded to 3.0-rc1. I guess I can help test 3.0 rather than use 2.2.

Should we upgrade the docs to reflect the correct instructions for Raspbian? I'd be happy to submit a PR if someone would accept.

**Comment by TomSweeneyRedHat** [comment]
We're always happy to take a PR @bfallik!

**Comment by lsm5** [comment]
Fixed by #343 . Closing, please reopen if it's still not resolved for you.

**Comment by bfallik** [comment]
thx @lsm5 !
